src/apis/sourceimages.js

```diff
@@ -203,6 +203,26 @@ export default (state) => {
   }
 
   /**
+   * Delete source images by its binary hash.
+   *
+   * ```js
+   * rokka.sourceimages.deleteWithBinaryHash('myorg', 'b23e17047329b417d3902dc1a5a7e158a3ee822a')
+   *   .then(function(result) {})
+   *   .catch(function(err) {});
+   * ```
+   *
+   * @authenticated
+   * @param  {string}  organization name
+   * @param  {string}  binaryHash   binary image hash
+   * @return {Promise}
+   */
+  sourceimages.deleteWithBinaryHash = (organization, binaryHash) => {
```

Comment:
nitpicking here, but can you name it `deleteByBinaryHash`? Somehow prefer that and also the docblock describes it as `Delete source images **by** its binary hash.`